src/services/common-service/common-resource-service.ts

```diff
@@ -35,6 +35,8 @@ export enum CommonResourceServiceError {
     UNIQUE_VIOLATION = 'UniqueViolation',
     OWNERSHIP_CYCLE = 'OwnershipCycle',
     MODIFYING_CONTAINER_REQUEST_FINAL_STATE = 'ModifyingContainerRequestFinalState',
+    UNIQUE_PUBLIC_KEY = 'UniquePublicKey',
```

Comment:
IMO errors generated by authorized-keys-service should be kept together, not in common-resource-service